lib/core/src/control/event_loop.rs

```diff
@@ -40,24 +46,26 @@ impl EventLoop {
     /// Init the event loop.
     fn init(self) -> Self {
         let data = Rc::downgrade(&self.rc);
-        let main = move |_| { data.upgrade().map(|t| t.borrow_mut().run()); };
+        let main = move |time_ms| { data.upgrade().map(|t| t.borrow_mut().run(time_ms)); };
         with(self.rc.borrow_mut(), |mut data| {
             data.main = Some(Closure::new(main));
-            data.run();
+            web::request_animation_frame(&data.main.as_ref().unwrap()).unwrap();
```

Comment:
What is the purpose of this change? I believe the previous version was working in the same way, but I might be missing something here :)

Reply:
@wdanilo, `data.run` now takes a time input and since we don't have this information at this moment, we call instead `request_animation_frame`.

Reply:
What is this time? Would it be ok if we pass `0` instead? Or the time doesnt start with 0?

Reply:
We can't pass 0 because it's fed with `performance.now()`. It returns the elapsed time, in milliseconds, from the beginning of the execution.